edgelet/Cargo.lock

```diff
@@ -108,7 +108,7 @@ dependencies = [
 [[package]]
 name = "aziot-certd-config"
 version = "0.1.0"
-source = "git+https://github.com/Azure/iot-identity-service?branch=main#e0b4f1c31112388ca79c6243787f1f0b70aa8721"
+source = "git+https://github.com/Azure/iot-identity-service?branch=main#7a222f5f0db2b7727f9e67a6eb66881f48fc63e3"
```

Comment:
when do we plan to move to release/1.2 branch of IIS?

Reply:
That's already done. I'm syncing master here.